src/ZendCodingStandard/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Copied from Squiz_Sniffs_Commenting_VariableCommentSniff
+ * with modifications:
+ *  - use suggested types from ZendCodingStandard\CodingStandard
+ *  - skip normal comments if there is one blank line after the comment
+ */
+
+namespace ZendCodingStandard\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\AbstractVariableSniff;
+use ZendCodingStandard\CodingStandard;
+
+class VariableCommentSniff extends AbstractVariableSniff
+{
+    /**
+     * @inheritDoc
+     */
+    public function processMemberVar(File $phpcsFile, $stackPtr)
+    {
+        $tokens = $phpcsFile->getTokens();
+        $ignore = [
+            T_PUBLIC,
+            T_PRIVATE,
+            T_PROTECTED,
+            T_VAR,
+            T_STATIC,
+            T_WHITESPACE,
+        ];
+
+        $commentEnd = $phpcsFile->findPrevious($ignore, $stackPtr - 1, null, true);
+        if ($commentEnd === false
+            || ($tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG
+                && $tokens[$commentEnd]['code'] !== T_COMMENT)
+        ) {
+            $phpcsFile->addError('Missing member variable doc comment', $stackPtr, 'Missing');
+            return;
+        }
+
+        if ($tokens[$commentEnd]['code'] === T_COMMENT) {
+            if ($tokens[$commentEnd]['line'] === $tokens[$stackPtr]['line'] - 1) {
+                $error = 'You must use "/**" style comments for a member variable comment';
+                $phpcsFile->addError($error, $stackPtr, 'WrongStyle');
+            }
+
+            return;
+        }
+
+        $commentStart = $tokens[$commentEnd]['comment_opener'];
+
+        $foundVar = null;
+        foreach ($tokens[$commentStart]['comment_tags'] as $tag) {
+            if ($tokens[$tag]['content'] === '@var') {
+                if ($foundVar !== null) {
+                    $error = 'Only one @var tag is allowed in a member variable comment';
+                    $phpcsFile->addError($error, $tag, 'DuplicateVar');
+                } else {
+                    $foundVar = $tag;
+                }
```

Comment:
It is copied form PHP_CodeSniffer with some changes described on top of the file.